spec/models/response_csv_spec.rb

```diff
@@ -125,7 +129,9 @@
       end
 
       responses = Response.order(:id)
-      expected = File.read(File.expand_path('../../expectations/response_csv/repeat_groups.csv', __FILE__))
+      uuids = responses.map(&:uuid)
+      shortcodes = responses.map(&:shortcode)
+      expected = ResponseCSVExpectationMaker.make_expectation_with_repeat_groups(uuids, shortcodes)
```

Comment:
So this would become `expected = response_csv_expectation(responses, repeat_groups: true)`

The `repeat_groups` named param would default to false.